openfe/protocols/openmm_rbfe/equil_rbfe_methods.py

```diff
@@ -570,25 +509,33 @@ def _default_settings(cls) -> RelativeLigandProtocolSettings:
 
         Returns
         -------
-        RelativeLigandProtocolSettings
+        Settings
           a set of default settings
         """
-        return RelativeLigandProtocolSettings(
-            system_settings=SystemSettings(
-                constraints='HBonds'
-            ),
-            topology_settings=TopologySettings(
-                forcefield={'protein': 'amber99sb.xml',
-                            'ligand': 'openff-2.0.0.offxml',
-                            'solvent': 'tip3p.xml'},
+        return settings.Settings(
+            settings_version=1,
+            forcefield_settings=settings.OpenMMSystemGeneratorFFSettings(),
+            thermo_settings=settings.ThermoSettings(
+                temperature=298.15 * unit.kelvin,
+                pressure=1 * unit.bar,
             ),
-            alchemical_settings=AlchemicalSettings(),
-            sampler_settings=SamplerSettings(),
-            barostat_settings=BarostatSettings(),
-            integrator_settings=IntegratorSettings(),
-            simulation_settings=SimulationSettings(
-                equilibration_length=2.0 * unit.nanosecond,
-                production_length=5.0 * unit.nanosecond,
+            protocol_settings=RelativeLigandProtocolSettings(
+                system_settings=SystemSettings(
+                    constraints='HBonds'
+                ),
+                topology_settings=TopologySettings(
+                    forcefield={'protein': 'amber99sb.xml',
+                                'ligand': 'openff-2.0.0.offxml',
+                                'solvent': 'tip3p.xml'},
```

Comment:
this is a good moment to update this:
```suggestion
                    forcefield={'protein': 'amber/ff14SB.xml',
                                'ligand': 'openff-2.0.0.offxml',
                                'solvent': 'amber/tip3p_standard.xml'
                                'ions': 'amber/tip3p_HFE_multivalent.xml',},
```

Reply:
Indeed those are the forcefield settings found under `OpenMMSystemGeneratorFFSettings`.